gulpfile.babel.js

```diff
@@ -51,11 +51,15 @@ gulp.task('doc', (cb) => {
         .option('-p, --private', 'Include private API documentation')
```

Comment:
For the benefit of Thorn developers who want a copy of the API documentation open to browse.